commitlint.config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+	parserPreset: 'conventional-changelog-conventionalcommits',
+	rules: {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 1 tab. (indent)](https://app.codacy.com/app/PeterNgTr/codeceptjs-bshelper/pullRequest?prid=4722635)